docs/src/app/components/pages/components/progress.jsx

```diff
@@ -72,6 +72,12 @@ const ProgressPage = React.createClass({
             header: 'optional',
             desc: 'Override the inline-styles of the progress\'s root element.',
           },
+          {
+            name: 'color',
```

Comment:
May be worse to add an example with this property in action for the Linear and Circular Progress.